README.md

```diff
@@ -62,6 +63,9 @@ the names differ you may define the desktop file with **sailjail** command's **-
 Use of absolute paths is required, except for the desktop file which must be located in
 _/usr/share/applications_ or _/etc/sailjail/applications_.
 
+The ExecDBus value is an optional command line which will be used to auto start the application
```

Comment:
"optional command line _argument_"?

Reply:
No, it's a complete command line. The original proposal was for an additional argument but that gets complicated pretty quickly.

The Exec entry can contain field codes (https://specifications.freedesktop.org/desktop-entry-spec/1.1/ar01s06.html) which at a minimum we'd have to strip to reuse it with another argument for the Exec entry in the service file. That then leads to the question of what if anything to do if an application has positional arguments and the defined argument shouldn't be tacked on the end, do we define our own field code for that?